rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -1063,10 +1063,10 @@ def test_edit_token_with_missing_information(database):
     Test that editing a token that already exists with missing information doesn't
     raise any error and the information is updated
     """
-    token_address = string_to_evm_address('0xf53AD2c6851052A81B42133467480961B2321C09')
+    token_address = string_to_evm_address('0xc37b40ABdB939635068d3c5f13E7faF686F03B65')
     peth = get_or_create_evm_token(
         userdb=database,
-        symbol='PETH',
+        symbol='CPETH',
```

Comment:
Why CPETH? Is it just any token that is supposed to be missing? Thenplease use a more exotic/random name.